agot-bg-game-server/src/client/game-state-panel/ResolveSingleMarchOrderComponent.tsx

```diff
@@ -196,6 +196,12 @@ export default class ResolveSingleMarchOrderComponent extends Component<GameStat
             return;
         }
 
+        if(this.plannedMoves.size == 0) {
+            if(!confirm("Do you want to remove your march order?")) {
```

Comment:
I'm really not a fan of `confirm` prompt but I guess we can keep them for the time being.

In a later PR, we can replace them with native prompts.